sharding-core/sharding-core-execute/src/main/java/org/apache/shardingsphere/core/execute/sql/execute/result/QueryResultMetaData.java

```diff
@@ -144,6 +157,9 @@ public boolean isCaseSensitive(final int columnIndex) throws SQLException {
      */
     public Optional<ShardingEncryptor> getShardingEncryptor(final int columnIndex) throws SQLException {
         String logicTable = getTableName(columnIndex);
```

Comment:
If the new function, `getTableName(columnIndex, sqlStatementContext)` is availiable, do you think this function is unneeded?